src/components/atoms/navbar.tsx

```diff
@@ -129,7 +129,7 @@ const styles = StyleSheet.create({
   left: {
     width: '25%',
     alignItems: 'flex-start',
-    paddingLeft: 12
+    paddingHorizontal: 12
```

Comment:
Not a big one, but for sure alters the initial intent.

Reply:
I'll re-assess this and the other one. Can't remember what the issue was here but I know I did these before discovering that `marginStart` was less prone to missbehave when nested than `paddingStart` so that might be a cleaner fix